packages/cli/lib/commands/create.js

```diff
@@ -96,12 +96,17 @@ function requestParams(argv, templates) {
 }
 
 async function updateTemplatesCache() {
-	const repos = await fetch(TEMPLATES_REPO_URL).then(r => r.json());
-	await fs.writeFile(
-		TEMPLATES_CACHE_FILENAME,
-		JSON.stringify(repos, null, 2),
-		'utf-8'
-	);
+	try {
+		const repos = await fetch(TEMPLATES_REPO_URL).then(r => r.json());
+		await fs.writeFile(
+			TEMPLATES_CACHE_FILENAME,
+			JSON.stringify(repos, null, 2),
+			'utf-8'
+		);
+	} catch (err) {
+		// eslint-disable-next-line
+		console.log(err);
```

Comment:
```suggestion
		console.log('failed to update template cache', err);
```

Reply:
Should maybe be `console.error` as well?

Reply:
should use utils?